portfolio/src/main/webapp/index.html

```diff
@@ -63,6 +63,13 @@ <h1>Omar Montiel's Portfolio</h1>
       <!-- Contact Section -->
       <div id="contact" class="section">
         <a href="resume.pdf" target="_blank">Resume</a>
+        <!-- Message Input Form -->
```

Comment:
Thanks for your comments!

Reply:
Welcome! Think I'm getting the hang of it😁